fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1472,6 +1474,9 @@ public void setEnableParallelMerge(boolean enableParallelMerge) {
     @VariableMgr.VarAttr(name = ENABLE_SHORT_CIRCUIT)
     private boolean enableShortCircuit = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_PREPARE_STMT)
+    private boolean enablePrepareStmt = false;
```

Comment:
If the default value is false, this should be cherry-picked to branch-3.2, otherwise, the system will downgrade when upgrading.
But if this is cherry-picked to version 3.2, this will make no back compatible when upgrading from 3.2.1 to 3.2.2.
So how do we handle is?

Reply:
the default value also can be true. just update/insert stmt need connect leader fe,  othersise will cause some error